app/views/application/_navbar.html.erb

```diff
@@ -23,13 +23,16 @@
           </li>
         <% end %>
 
-        <% if current_user.roles.rvt_member? %>
+        <% if current_user.roles.rvt_member? || policy(:application).manage_SVIE? %>
```

Comment:
Here the latter statement already includes the check for rvt member roles, right? So I think we only need the policy check.